modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -12,6 +12,11 @@ You can activate the following feature set by using the `FeatureGate` CR:
 
 * `TechPreviewNoUpgrade`. This feature set is a subset of the current Technology Preview features. This feature set allows you to enable these tech preview features on test clusters, where you can fully test them, while leaving the features disabled on production clusters. Enabling this feature set cannot be undone and prevents minor version updates. This feature set is not recommended on production clusters.
 +
+[WARNING]
+====
+Enabling the `TechPreviewNoUpgrade` feature set cannot be undone and prevents minor version updates. You should not enable this feature set on production clusters.
```

Comment:
Up to you though! Not really a big difference. 
```suggestion
Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. You should not enable this feature set on production clusters.
```

Reply:
Sounds good, I'll add the extra specificity. Thanks!